parm/post_avblflds.xml

```diff
@@ -7709,6 +7709,469 @@
        <scale>6.0</scale>
     </param>
 
+    <param>
```

Comment:
@hsinmulin-NOAA May I ask why you assign the new field ID starting at 1185 instead 994?